test/run.sh

```diff
@@ -68,19 +68,19 @@ cd "${SCRIPTDIR}" || (echo "Did not find ${SCRIPTDIR}" 1>&2; exit 1)
 TESTS="$*"
 # if TESTS is not set - run the standard suite.
 if [ -z "${TESTS}" ]; then
-    TESTS="./suites/standard"
+    TESTS="./suites/standard1 ./suites/standard2"
 fi
 
 set -x
 if ${DRYRUN}; then
     # shellcheck disable=SC2086
-    "${RF_BINARY}" \
+    eval "${RF_BINARY}" \
```

Comment:
Why do we want eval here?

Reply:
Because we need to expand var `TEST`, otherwise we are passing all together as param 
`'suites/standard1 ./suites/standard2'`
instead of 
`'suites/standard1'` and `'./suites/standard2'` 
getting this error:

[ ERROR ] Parsing 'suites/standard1 ./suites/standard2' failed: File or directory to execute does not exist.

Reply:
remove eval and use TEST as an array var